packages/babel-traverse/src/types.ts

```diff
@@ -11,8 +11,6 @@ export type Visitor<S = {}> = VisitNodeObject<S, t.Node> &
   } &
   {
     [K in keyof VirtualTypeAliases]?: VisitNode<S, VirtualTypeAliases[K]>;
-  } & {
-    [k: string]: VisitNode<S, t.Node>;
```

Comment:
remove indexed signatures here since it overrides the TraverseOptions (scope, noScope, denylist)

as result, the key like `"AssignmentExpression|Declaration|VariableDeclarator"` will cause an error